modules/roxctl-central-generate-openshift.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * command-reference/roxctl-central.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="roxctl-central-generate-openshift_{context}"]
+= roxctl central generate openshift
+
+Generate the required YAML configuration files to deploy Central in a {osp} cluster.
+
+.Usage
+[source,terminal]
+----
+$ roxctl central generate openshift [flags]
+----
+
+.Options
+[cols="6,3",options="header"]
+|===
+|Option |Description
+
+|`--central-db-image string`
+|Specify the Central database image that you want to use. If not specified, a default value is created corresponding to the `--image-defaults`.
+
+|`--declarative-config-config-maps strings`
+|Specify a list of configuration maps that you want to add as declarative configuration mounts in Central.
+
+|`--declarative-config-secrets strings`
+|Specify a list of secrets that you want to add as declarative configuration mounts in Central.
+
+|`--enable-telemetry`
+|Specify whether you want to enable telemetry. The default value is `false`.
+
+|`--image-defaults string`
+|Specify the default settings for container images. It controls the repositories from which the images are downloaded, the image names and the format of the tags. The default value is `development_build`.
+
+|`--istio-support version`
+|Generate deployment files that support the specified Istio version. Valid values include `1.0`, `1.1`, `1.2`, `1.3`, `1.4`, `1.5`, `1.6`, and `1.7`.
+
+|`--lb-type load balancer type`
+|Specify the method of exposing Central. Valid values include `route`, `lb`, `np` and `none`. The default value is `none`.
+
+|`-i`, `--main-image string`
+|Specify the main image that you want to use. If not specified, a default value corresponding to `--image-defaults` is used.
+
+|`--offline`
+|Specify whether you want to run {product-title-short} in offline mode, avoiding a connection to the Internet. The default value is `false`.
+
+|`--openshift-monitoring false|true|auto[=true]`
```

Comment:
This line isn't displaying properly - it's being split into two rows, looks like
![Screenshot 2024-02-27 at 4 25 53 PM](https://github.com/openshift/openshift-docs/assets/96192119/25e2fc51-dc7b-4085-9f01-654b66b5da78)